protocol/avdl/keybase1/metadata.avdl

```diff
@@ -41,10 +41,15 @@ protocol metadata {
          KeyBundle ReaderBundle;
   }
 
+  record LockContext {
+    long    requireLockID;
```

Comment:
Sure, a helper function that takes in a `[]byte` and returns a `LockID` seems good to me.  If it just does simple truncation of the byte array to 8 bytes, it should probably document that the first 8 bytes needs to be as uniquely identifying as possible.  Of course if it does a hash instead, that shouldn't matter...